pkg/arvo/sys/vane/clay.hoon

```diff
@@ -3189,6 +3353,10 @@
   ++  set-zest                                          ::  [goad] <
     |=  liv=zest
     =?  liv  =(%base syd)  %live
+    ?:  ?&  =(%live liv)
```

Comment:
should this crash on failure too?

Reply:
nvm, this does crash on failure -- my second reading comprehension error of the day